main.go

```diff
@@ -59,6 +61,8 @@ func init() {
 	utilruntime.Must(placementv1.AddToScheme(scheme))
 	utilruntime.Must(glancev1.AddToScheme(scheme))
 	utilruntime.Must(cinderv1.AddToScheme(scheme))
+	utilruntime.Must(rabbitmqv1beta1.AddToScheme(scheme))
+	utilruntime.Must(cinderv1.AddToScheme(scheme))
```

Comment:
-1: This is a duplicate of L63, so we should only leave one of them, right?

Reply:
Yes, will remove.